dotnet/Liberator.Driver/RatDriverMethods.cs

```diff
@@ -364,6 +357,50 @@ public void ClickLinkAndWait(By locator, [Optional, DefaultParameterValue(true)]
             }
         }
 
+        /// <summary>
+        /// Clicks on a link and waits for a new page to be loaded that contains a specified URL or part URL
+        /// </summary>
+        /// <param name="element">The element on which to click</param>
+        /// <param name="url">The partial URL to be waited for</param>
+        /// <param name="waitForTarget">(Optional parameter) Whether to wait for the cliackability of the target element</param>
+        public void ClickLinkAndWaitForUrl(IWebElement element, string url, [Optional, DefaultParameterValue(true)] bool waitForTarget)

```

Comment:
Again, we dont need the wait for target bool as we are not using it

Reply:
Changed